### PR TITLE
Kafka Connector: confluent cloud usage, links to msk

### PR DESCRIPTION
**Pull request by mshustov**

### CODE REVIEW

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 1]
```diff
@@ -754,13 +754,26 @@ Navigate to “Connectors” -> “Add Connector” and use the following settin
...
+
 You should be all set!
 
 ##### Known Limitations
+* Custom Connectors must use public internet endpoints. Static IP addresses aren't supported.
 * You can override some Custom Connector properties. See the fill [list in the official documentation.](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-manage.html#override-configuration-properties)
 * Custom Connectors are available only in [some AWS regions](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-fands.html#supported-aws-regions)
 * See the list of [Custom Connectors limitations in the official docs](https://docs.confluent.io/cloud/current/connectors/bring-your-connector/custom-connector-fands.html#limitations)
-
+#### Installing on AWS MSK
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:775]: @rfraposa why did you merge all the Kafka-related docs into a single page in this PR https://github.com/ClickHouse/clickhouse-docs/pull/850? Do you know if it is done accidentally?

**Comment by robot-clickhouse** [comment]
This is an automated comment for commit 13a69ad5023325ff2aee436f6f83112a5d532cf6 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1497/13a69ad5023325ff2aee436f6f83112a5d532cf6/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🔴 failure</td></tr>
</table>

File: docs/en/integrations/data-ingestion/kafka/index.md [hunk 2]
```diff
@@ -754,13 +754,26 @@ Navigate to “Connectors” -> “Add Connector” and use the following settin
   "value.converter.schemas.enable": "false"
 }
 ```
+#### Specify the connection endpoints
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/index.md:757]: This step is missing from the docs

**Comment by mzitnik** [comment]
@mshustov do we have screenshots on the BYOC setup?

**Comment by mshustov** [comment]
@mzitnik there are some https://clickhouse.com/docs/en/integrations/kafka#installing-on-confluent-cloud

File: docs/en/integrations/data-ingestion/kafka/index.md (hunk 1 above)
Comment (rfraposa) [reply docs/en/integrations/data-ingestion/kafka/index.md:775]: No - it was deliberate. At the time, Kafka (and quite a few other topics) were spread across multiple pages. It was an attempt to make the docs easier to navigate and find what you need - by reducing the number of docs pages and consolidating those topics to a single page.

This was one of several PRs that condensed topics to a single page.

Comment (mshustov) [reply docs/en/integrations/data-ingestion/kafka/index.md:775]: @rfraposa  I use the page regularly and find it unfriendly: it's almost impossible to search for something on the page due to its size. I want to revert these changes to the structure we introduced in https://github.com/ClickHouse/clickhouse-docs/pull/740.

**Review by mzitnik — APPROVED** [review]
LGTM

File: docs/en/integrations/data-ingestion/kafka/index.md (hunk 1 above)
Comment (rfraposa) [reply docs/en/integrations/data-ingestion/kafka/index.md:775]: Probably a conversation to have with @justindeguzman instead of me.

Comment (justindeguzman) [reply docs/en/integrations/data-ingestion/kafka/index.md:775]: @mshustov it'd be good to sync with you and @Ryado on this for the Kafka docs rewrites

**Pull request merged**